Test/task3.c

```diff
@@ -0,0 +1,106 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+void readAllTheComments(FILE *file, char *data[], int linesRead);
+
+
+int main()
+{
+    FILE *file1 = fopen("allTheLinesAreComments.txt", "r");
+    FILE *file2 = fopen("NoComments.txt", "r");
+    FILE *file3 = fopen("commetsAndNoCommentsAreAlternating.txt", "r");
+
+    char *data1[20] = { 0 };
+
+    int linesRead1 = 0;
+
+    while(!feof(file1))
+    {
+        char *buffer = malloc(sizeof(char) * 100);
+        fgets(buffer, 100, file1);
+        
+        if(buffer[0] == ';')
+        {
+            data1[linesRead1] = buffer;
+            linesRead1++;
+        }
+
+    }
```

Comment:
```suggestion
        }
    }
```
Между `}` пустая строка тоже не ставится